api/permissions_requests_handlers.go

```diff
@@ -0,0 +1,96 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/topfreegames/Will.IAM/models"
+	"github.com/topfreegames/Will.IAM/usecases"
+	"github.com/topfreegames/extensions/middleware"
+)
+
+func permissionsRequestsCreateHandler(
+	prsUC usecases.PermissionsRequests,
+) func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		l := middleware.GetLogger(r.Context())
+		pr := &models.PermissionRequest{}
+		if err := readBodyTo(r, pr); err != nil {
+			l.WithError(err).Error("failed to read body")
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+		saID, _ := getServiceAccountID(r.Context())
```

Comment:
Maybe add a comment explaining why this is safe to swallow, or create a `mustGetServiceAccountID` method that `log.Fatal`s on error?

Same for others.

Reply:
_ is a bool, if it's not present saID will be "" and other functions will fail, resulting in 500. Seems less drastic than Fatal. What do you think?

Reply:
Sounds good. If we're confident it's true, than I'd rather Fatal than swallow, but checking is even better.